workbench-core/authorization/src/caslAuthorizationPlugin.ts

```diff
@@ -3,18 +3,20 @@
  *  SPDX-License-Identifier: Apache-2.0
  */
 
-import { Ability, AbilityBuilder, ForbiddenError as CASLForbiddenError } from '@casl/ability';
+import { Ability, AbilityBuilder, ForbiddenError as CASLForbiddenError, subject } from '@casl/ability';
 import AuthorizationPlugin from './authorizationPlugin';
+import { IdentityPermission } from './dynamicAuthorization/dynamicPermissionsPluginInputs';
 import { ForbiddenError } from './errors/forbiddenError';
-import Operation from './operation';
+import { DynamicOperation, Operation } from './operation';
 import Permission from './permission';
+
 /**
  * {@link https://github.com/stalniy/casl | CASL } Authorization Plugin.
  *
  */
 export default class CASLAuthorizationPlugin implements AuthorizationPlugin {
   public async isAuthorized(userPermissions: Permission[], operations: Operation[]): Promise<void> {
-    const ability: Ability = this._defineAbilitiesFor(userPermissions);
+    const ability: Ability = this._defineAbilitiesForUserPermissions(userPermissions);
```

Comment:
Not sure if the type is declared for easy reading, but you don't need to declare it since the return type is declared on the function you're calling.

Reply:
It was declared for ease of reading